models/integration_pd.py

```diff
@@ -1,20 +1,16 @@
+import json
 from typing import List, Optional
 from pydantic import BaseModel, root_validator, validator
 
-from tools import session_project, rpc_tools
+from tools import session_project, rpc_tools, VaultClient
 from pylon.core.tools import log
 from ...integrations.models.pd.integration import SecretField
 
 
-CAPATIBILITIES_MAP = {
-    'completion':
-        ['gpt-3.5-turbo-instruct', 'babbage-002', 'davinci-002'],
-    'chat_completion':
-        ['gpt-4', 'gpt-4-0613', 'gpt-4-32k', 'gpt-4-32k-0613', 'gpt-3.5-turbo',
-         'gpt-3.5-turbo-0613', 'gpt-3.5-turbo-16k', 'gpt-3.5-turbo-16k-0613'],
-    'embeddings':
-        ['text-embedding-ada-002', 'davinci', 'curie', 'babbage', 'ada']
-}
+def get_capabilities_map():
```

Comment:
I think you should accept projec_id and initialize project aware vault so the secret responsible for mapping can be overriden in project